ros/api/main.py

```diff
@@ -8,4 +8,4 @@
 initialize_routes(api)
 
 if __name__ == "__main__":
-    app.run(host='0.0.0.0', port=8080)
+    app.run(host='0.0.0.0', port=8000)
```

Comment:
Do you have any apps that depend on your service? Just making sure before you change the port here.

Reply:
Yeah so we have all these services running for us: https://github.com/RedHatInsights/ros-backend/blob/main/scripts/docker-compose.yml